### PR TITLE
PipsPager control visual updates

### PR DESCRIPTION
**Pull request by marksfoster**
PipsPager control visual updates.

Updates styles to POR.

Screenshots:

Light Theme
image
image

Dark Theme
image
image

HC
image

### CODE REVIEW

File: dev/PipsPager/PipsPager_themeresources.xaml [hunk 1]
```diff
@@ -5,184 +5,305 @@
...
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBackgroundSelected" ResourceKey="ControlFillColorTransparentBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBackgroundDisabled" ResourceKey="ControlFillColorTransparentBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBorderBrush" ResourceKey="ControlFillColorTransparentBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBorderBrushPointerOver" ResourceKey="ControlFillColorTransparentBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBorderBrushPressed" ResourceKey="ControlFillColorTransparentBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBorderBrushSelected" ResourceKey="ControlFillColorTransparentBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBorderBrushDisabled" ResourceKey="ControlFillColorTransparentBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorForeground" ResourceKey="ControlAAFillColorDefaultBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorForegroundPointerOver" ResourceKey="TextFillColorSecondaryBrush" />
```
[the remark this answers is not in the document]
Comment (marksfoster) [reply dev/PipsPager/PipsPager_themeresources.xaml:19]: Yep, same color for hover and pressed is correct

File: dev/PipsPager/PipsPager_themeresources.xaml [hunk 2]
```diff
@@ -5,184 +5,305 @@
...
 
         <ResourceDictionary x:Key="HighContrast">
-            <StaticResource x:Key="PipsPagerSelectionIndicatorForeground" ResourceKey="SystemControlForegroundAccentBrush" />
-            <StaticResource x:Key="PipsPagerPageNavigationButtonBackgroundPointerOver" ResourceKey="SystemControlHighlightListLowBrush" />
-            <StaticResource x:Key="PipsPagerPageNavigationButtonBackgroundPressed" ResourceKey="SystemControlHighlightListMediumBrush" />
-            <StaticResource x:Key="PipsPagerPageNavigationButtonBackgroundDisabled" ResourceKey="SystemControlTransparentBrush" />
-            <StaticResource x:Key="PipsPagerPageNavigationButtonForegroundPointerOver" ResourceKey="SystemControlHighlightAltBaseHighBrush" />
-            <StaticResource x:Key="PipsPagerPageNavigationButtonForegroundPressed" ResourceKey="SystemControlHighlightAltBaseHighBrush" />
-            <StaticResource x:Key="PipsPagerPageNavigationButtonForegroundDisabled" ResourceKey="SystemControlDisabledBaseMediumLowBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBackground" ResourceKey="SystemControlBackgroundListLowBrush" />
```
[the remark this answers is not in the document]
Comment (marksfoster) [reply dev/PipsPager/PipsPager_themeresources.xaml:68]: Will do thanks

**Comment by tashatitova** [comment]
Consider adding a HC screenshot

File: dev/PipsPager/PipsPager_themeresources.xaml [hunk 3]
```diff
@@ -5,184 +5,305 @@
...
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBackgroundPointerOver" ResourceKey="SystemColorHighlightColorBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBackgroundPressed" ResourceKey="SystemColorHighlightColorBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBackgroundSelected" ResourceKey="SystemColorHighlightColorBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBackgroundDisabled" ResourceKey="SystemColorButtonFaceColorBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBorderBrush" ResourceKey="SystemColorButtonFaceColorBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBorderBrushPointerOver" ResourceKey="SystemColorHighlightColorBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBorderBrushPressed" ResourceKey="SystemColorHighlightColorBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBorderBrushSelected" ResourceKey="SystemColorHighlightColorBrush" />
+            <StaticResource x:Key="PipsPagerSelectionIndicatorBorderBrushDisabled" ResourceKey="SystemColorButtonFaceColorBrush" />
```
Comment (tashatitova) [inline dev/PipsPager/PipsPager_themeresources.xaml:77]: Shouldn't this be GreayText for disabled?

Comment (marksfoster) [reply dev/PipsPager/PipsPager_themeresources.xaml:77]: There is no border on selected item when disabled. Drawing from HC spec:

image

Comment (tashatitova) [reply dev/PipsPager/PipsPager_themeresources.xaml:77]: Still I'd point it to disabled color for consistency, shouldn't be button face.

**Review by tashatitova — APPROVED** [review]

File: dev/PipsPager/PipsPager_themeresources.xaml (hunk 3 above)
Comment (marksfoster) [reply dev/PipsPager/PipsPager_themeresources.xaml:77]: It wouldn't be to spec.

**Review by beervoley — APPROVED** [review]

**Pull request merged**